src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -1047,4 +1040,49 @@ internal override void ForceComplete(SourceLocation locationOpt, CancellationTok
             var unused = this.ExplicitDefaultConstantValue;
         }
     }
+
+    internal sealed class SourceComplexParameterSymbolWithCustomModifiers : SourceComplexParameterSymbol
```

Comment:
This happens when we override a method that has modifiers. The modifiers are inherited from the overriden method.